.github/lock.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for Lock Threads - https://github.com/dessant/lock-threads
+
+# Number of days of inactivity before a closed issue or pull request is locked
+daysUntilLock: 1
```

Comment:
I wonder if we should set it to 7 days? Immediate feedback might be useful, and 7 days is the time the beta lasts, which is what usually attracts bad comments (because we do a release that links all the posts)

Reply:
I started writing a comment on which I agreed and suggest to prolong it to 21 in that case to cover the whole release period. Remove it, started over again.

I think it should be issues, not comments on PR. Issues can be milestoned and marked as a beta issue. Those are trackable. Whatever happens on a closed PR is not trackable in any way.

I'm happy to adjust, but please consider the above.

Reply:
ok